src/mixins/normalize.js

```diff
@@ -281,13 +280,11 @@ function mergeRules(baseRules: Object, additionalRules: Object) {
  * // CSS as JS Output
  *
  * html {
- *   font-family: sans-serif,
- *   line-height: 1.15,
- *   -ms-text-size-adjust: 100%,
- *   -webkit-text-size-adjust: 100%,
+ *   fontFamily: sans-serif,
```

Comment:
I'm confused about this one. If it's JS output it `sans-serif` should be wrapped in String?!?